<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>692</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>600</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>329</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>43</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset theme="application-exit">
      <normaloff>icons/exit.png</normaloff>icons/exit.png</iconset>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionru"/>
    <addaction name="actionen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit" resource="resources.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>
     <normalon>icons/exit.png</normalon>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </action>
  <action name="actionru">
   <property name="text">
    <string>ru</string>
   </property>
  </action>
  <action name="actionen">
   <property name="text">
    <string>en</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
